src/PrestaShopBundle/Form/Admin/Improve/Design/Pages/CmsPageType.php

```diff
@@ -121,7 +131,17 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                     ],
                 ],
             ])
+            ->add('seo_preview', FreeType::class, [
+                'label' => $this->trans('SEO preview', 'Admin.Global'),
+                'help' => $this->trans('Here is a preview of how your page will appear in search engine results.', 'Admin.Global'),
+                'view_data' => [
+                    // @todo: should generate actual URL
```

Comment:
I add a WIP because of this

Reply:
@PierreRambaud you are faster than Chuck Norris, reviewed PR before I even submitted it. :open_mouth:

Reply:
Done.